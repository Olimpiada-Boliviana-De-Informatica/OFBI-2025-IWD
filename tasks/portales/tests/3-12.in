85
374172779 857563025 393746492 550666961 277665402 345072720 203854246 983266945 637861512 163020268 56560413 571609922 581431503 437496497 402605542 651556985 315916350 161663167 98921786 215399073 267096312 487506570 321300677 967313350 90241747 174138200 797326121 742746123 21932506 19611590 984997497 647499812 693454539 661218372 894357182 749059549 260961690 426942356 655748726 712343697 227113964 774397775 384553232 781851059 471626653 57701472 796948849 775770441 654635323 460464518 263012466 836144747 835011208 714895319 993420127 313475017 475005691 317545784 574392441 144482851 485079616 761301733 165294965 750218446 237857460 450714096 638327804 723311158 526130666 656399801 62164009 940971094 456169768 57962619 993778446 124321089 84079082 207242688 514477871 308100509 403309662 849682663 273082628 571389219 377966567