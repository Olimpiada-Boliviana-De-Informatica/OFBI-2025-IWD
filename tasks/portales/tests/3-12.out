-1
374172779
374172779
393746492
-1
277665402
-1
857563025
550666961
-1
-1
550666961
571609922
393746492
393746492
637861512
277665402
56560413
56560413
203854246
215399073
437496497
315916350
857563025
56560413
163020268
651556985
651556985
-1
-1
983266945
637861512
651556985
651556985
857563025
742746123
215399073
402605542
651556985
693454539
215399073
749059549
374172779
774397775
437496497
56560413
781851059
774397775
651556985
437496497
260961690
797326121
797326121
712343697
984997497
277665402
471626653
315916350
571609922
98921786
475005691
749059549
163020268
749059549
227113964
437496497
637861512
714895319
487506570
655748726
57701472
894357182
450714096
57701472
993420127
98921786
62164009
203854246
487506570
277665402
402605542
836144747
267096312
550666961
374172779
